### ISSUE TITLE
Missing link in `vkCmdPipelineBarrier` page

### ISSUE THREAD

**Issue by nicebyte** [KhronosGroup/Vulkan-Docs#1299]
The "Valid Usage" section on `vkCmdPipelineBarrier` [doc page](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkCmdPipelineBarrier.html) points to a "table of supported access types" as well as a "table of supported pipeline stages".  Both link to an anchor within the same page, but the tables themselves are missing.

**Comment by oddhack** [comment]
Good catch. That's going to be tricky to deal with. We rewrite links inside the reference page bodies, but those are coming from a separate asciidoctor file that's included inside the VU section, and getting those rewritten will be a pain. Unlikely to be fixed in the immediate future but I'll put it on the list.

**Comment by holtowd** [comment]
Had it been already solved? Because when I click on those links it gets me to the tables
